components/YJzc7NwGrLmKp6gfZJh7Xs.yml

```diff
@@ -0,0 +1,26 @@
+name: response_selector
+display_name: Ranking-based Response Selector
```

Comment:
you do not need a new card for this response selector because it is already existing